templates/dashboard.html.twig

```diff
@@ -66,9 +66,9 @@
                                     <v-spacer></v-spacer>
                                     <v-btn
                                             color="secondary" flat small
-                                            :href="'https://packagist.org/packages/' + recipe.officialPackageName"
+                                            :href="getRepoPackagistUrl(recipe, '{{ private_packagist_url }}')"
```

Comment:
Passing the `private_packagist_url` into this getter also doesn't seem right to me. Just passing something from the .env via config to twig in order to inject it into a specific vue component for one method feels a bit hacky. One could pass the private packagist url as a prop to the dashboard component. This could then be used in the computed property for the  `packagistUrl ` in `dashboard.js`.